rest-servlet/src/main/java/org/hawkular/inventory/rest/RestEnvironments.java

```diff
@@ -57,14 +60,16 @@
 
     @GET
     @Path("/{tenantId}/environments")
-    @ApiOperation("Returns all environments under given tenant.")
+    @ApiOperation("Returns all environments under given tenant. Accepts paging query parameters.")
```

Comment:
yeah, when I thought I'd have to do the `@QueryParam` dance like 30 times, I was desperate to find another way ;)